src/WinRPM.jl

```diff
@@ -349,7 +350,7 @@ end
 
 install(pkg::AbstractString, arch::AbstractString=OS_ARCH; yes = false) = install(select(lookup(pkg, arch),pkg); yes = yes)
 
-function install(pkgs::Vector{ASCIIString}, arch = OS_ARCH; yes = false)
+function install(pkgs::Vector{String}, arch = OS_ARCH; yes = false)
```

Comment:
I don't think this will work on 0.4 or earlier, `Compat.ASCIIString` is less risky